datajunction-server/datajunction_server/api/health.py

```diff
@@ -44,12 +45,14 @@ async def database_health(session: Session) -> HealthcheckStatus:
             HealthcheckStatus.OK if result == (1,) else HealthcheckStatus.FAILED
         )
         return health_status
-    except Exception:  # pylint: disable=broad-except
-        return HealthcheckStatus.FAILED
+    except Exception:  # pylint: disable=broad-except  # pragma: no cover
```

Comment:
Similar question here. Not a problem to use no cover when needed, I'm just curious why the change from sqlmodel -> sqlalchemy would remove this coverage since I don't see any related changes in the tests.

Reply:
This one actually isn't covered -- I think it's related to how the unit test mocks away the execute: in the SQLModel case the mock caused `session.exec(...)` to raise an error, which would hit this except clause, but in SQLAlchemy, it doesn't, so it just returns a failed healthcheck.

Reply:
So is it safe to remove the try except?

Reply:
I don't think we should because if the session.execute call fails, the healthcheck should fail rather than returning a 500. We can add a test case for this.